modules/cms/classes/CmsObjectCollection.php

```diff
@@ -35,14 +36,38 @@ public function withComponent($components, $callback = null)
         });
     }
 
+    /**
+     * Returns objects whose properties match the supplied value.
+     *
+     * This is a wrapper for our custom `applyWhereFilter` method, as the signature of this method changed in Laravel 6.
+     *
+     * @param  string  $key
+     * @param  mixed  $operator
+     * @param  mixed  $value
+     * @return static
+     */
+    public function where($key, $operator = null, $value = null)
```

Comment:
@bennothommo is this a breaking change technically? How does this affect the Halycon datasources?

Reply:
@LukeTowers I believe it's not a breaking change - the new method signature is technically less restrictive than the old signature. I've made that particular method a wrapper which replicates the restrictions of the old method and forwards the call on to a new method which is essentially our old method. `CmsObjectCollection` doesn't appear to be extended by another class, so I believe it should be fine.

Reply:
@bennothommo what used to call this method though? Whatever called this method would still be sending the old signature. Additionally, it seems like this is misleading, since we haven't actually changed the behavior of the internal filter to take the operator into account.

Reply:
@LukeTowers I think it's mainly called when using a query against Halcyon objects. The above change will work with the old signature. It would only fail if someone wants to use the *new* signature for Halcyon objects (ie. adding an operator to `where` queries) - as far as I can tell, that's not happening as it's never supported that previously.

I felt that since Halcyon objects are something completely unique to October, it would be safe to keep the behaviour as is.

Reply:
@bennothommo do we have to change the parameter names in order to pass the matching signature check? If not I'd prefer to keep the old names even if it means we have to change their defaults. 

Either way there should be a comment about how it doesn't actually match the new laravel signature

Reply:
@LukeTowers it appears we can keep the property names, but they have to remain with `null` as default in order to match the signature from Laravel. I've changed the method around to suit, and added a comment with the details of the change.